schema/snapcraft.yaml

```diff
@@ -331,27 +331,50 @@ properties:
               required: [snap, prime]
               validation-failure:
                   "Parts cannot contain both 'snap' and 'prime' keywords."
+
           # Make sure prepare/override-build are mutually exclusive
           - not:
               type: object
               required: [prepare, override-build]
               validation-failure:
-                  "Parts cannot contain both 'prepare' and 'override-build'
-                  keywords. Use 'override-build'."
+                  "Parts cannot contain both 'prepare' and 'override-*'
+                  keywords. Use 'override-build' instead of 'prepare'."
```

Comment:
Yeah that's kind of the point. I was asked to ensure that one cannot use `prepare`/`build`/`install` (i.e. the old-style scriptlets) with ANY of the new-style scriptlets, which means all of the `override-*` keywords, whatever they may be. It felt weird to say "parts cannot contain both `prepare` and `override-pull`, use `override-build` instead." Instead I'm trying to be explicit and say "hey, these two words cannot cross. You should be using `override-build` instead of the old stuff."